stonesoup/updater/alphabeta.py

```diff
@@ -0,0 +1,115 @@
+    :math:`0 < \alpha < 1` and :math:`0 < \beta \leq 2`. Colloquially, the larger the values of the
+    parameters, the more influence the measurements have over the transition model; :math:`\beta`
+    is usually much smaller than :math:`\alpha`.
+
+    As the prediction is just the application of a constant velocity model, there is no
+    :math:`\alpha-\beta` predictor provided in Stone Soup. It is assumed that the predictions
+    passed to the hypothesis have been generated by a constant velocity model. Any application of a
+    control model is also assumed to have taken place during the prediction stage.
+
+    This class assumes the velocity is in units of the length per second. If different units are
+    required, scale the prior appropriately.
+
+    The measurement model used should be linear and a measurement model such that it provides a
+    'mapping' to :math:`p` via the :attr:`mapping` tuple and a binary measurement matrix which
+    returns :math:`p`. This isn't checked.
+
+    """
+
+    alpha: float = Property(doc="The alpha parameter. Controls the weight given to the "
+                                "measurements over the transition model.")
+    beta: float = Property(doc="The beta parameter. Controls the amount of variation allowed in "
+                               "the velocity component.")
+
+    vmap: np.ndarray = Property(default=None, doc="Binary map of the velocity elements in the"
+                                                  "state vector. If left default, the class will"
+                                                  "assume that the velocity elements interleave "
+                                                  "the position elements in the state vector.")
+
+    def predict_measurement(self, prediction, **kwargs):
+        """Return the predicted measurement
+
+        Parameters
+        ----------
+        prediction : :class:`~.StatePrediction`
+            The state prediction
+
+        Returns
+        -------
+         : :class:`~.StateVector`
+            The predicted measurement
+        """
+        pred_meas = self.measurement_model.matrix(**kwargs) @ prediction.state_vector
+        return MeasurementPrediction.from_state(prediction, pred_meas)
+
+    def update(self, hypothesis, time_interval, **kwargs):
+        """Calculate the inferred state following update
+
+        Parameters
+        ----------
+        hypothesis : :class:`~.Hypothesis`
+            A hypothesis associates a measurement with a prediction
+        time_interval : :class:`~.timedelta`
+            The time interval over which the prediction has been made.
+
+        Returns
+        -------
+         : :class:`~.StateUpdate`
+            The updated state
+        """
+        out_statevector = hypothesis.prediction.state_vector.copy()
+        pred_meas = self.predict_measurement(hypothesis.prediction)
+
+        pmap = np.array(self.measurement_model.mapping)
```

Comment:
I'm not sure this is always true, as some models may map velocity.  I think you could add `pmap` as a `Property` similar to `vmap`, defaulting to `None` and use this as the default.

Reply:
It should always be true in the alpha-beta filter, as you'd only choose a linear measurement model which returned position. I suppose there might be possible to construct a linear measurement model which doesn't have mapping, but it wouldn't inherit from `MeasurementModel`.

But could still do your suggestion.- it would just introduce an unused default.